air-script/tests/variables/variables.air

```diff
@@ -14,7 +14,7 @@ periodic_columns:
     k0: [1, 1, 1, 1, 1, 1, 1, 0]
 
 boundary_constraints:
-    enf a.first = stack_inputs[0]
+    enf a.first = 0
```

Comment:
I changed this to keep the public inputs usage in the public inputs test. Once we add variables for the boundary constraints, we can test them here as well